app/controllers/api/properties_controller.rb

```diff
@@ -1,6 +1,5 @@
 module Api
   class PropertiesController < Api::ApiController
-
```

Comment:
Actually by doing that you introduced more noise to the history. Even though the file seems untouched, the history say it changed twice now.

Reply:
AHHHHHHHHHHHHHHHH